master-firmware/protobuf/beacons.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto2";
+
+import "nanopb.proto";
+import "Timestamp.proto";
+
+message AlliedPosition
+{
+    option (nanopb_msgopt).msgid = 2;
+    required Timestamp timestamp = 1;
+    required float x = 2;
+    required float y = 3;
+}
+
+message BeaconSignal {
```

Comment:
the more generic concept is `RangeBearing`: 
- range = distance
- bearing = angle
this covers more than our proximity beacon (e.g. lidar data).

Reply:
So for range I can reuse the `Range` message, what do you think ?